library/crates/BUILD.bazel

```diff
@@ -156,6 +165,15 @@ alias(
     ],
 )
 
+alias(
+    name = "rustix",
+    actual = "@raze__rustix__0_35_9//:rustix",
+    tags = [
+        "cargo-raze",
+        "manual",
+    ],
+)
+
```

Comment:
`update.sh` wasn't apparently run after the last change in the repo, so this is added here.